.gitignore

```diff
@@ -6,4 +6,3 @@ yarn-error.log
 .stylelintcache
 .npmrc
 public
-dist
```

Comment:
Are you sure this is needed? Can it be built during the release? See https://github.com/bethinkpl/design-system/blob/master/.github/workflows/build.yml